src/download_labels.py

```diff
@@ -46,42 +48,33 @@ def way(self, w):
         self.extract_way_type(w)
 
     def extract_if_tennis_court(self, w):
-      name = ''
       is_tennis = False
       for tag in w.tags:
         if tag.k == 'sport' and 'tennis' == tag.v:
           is_tennis = True
-        if tag.k == 'name':
-          name = tag.v
 
       if not is_tennis:
         return
 
-      way_dict = {
-            'uid': w.uid,
-            'ends_have_same_id': w.ends_have_same_id(),
-            'id': w.id,
-            'tags':[]}
+      way_dict = {'uid': w.uid, 'ends_have_same_id': w.ends_have_same_id(), 'id': w.id, 'tags': []}
+
       for tag in w.tags:
         way_dict['tags'].append((tag.k, tag.v))
 
       self.add_linestring(w, way_dict)
 
     def extract_way_type(self, w):
       should_extract = False
-      name = ''
       way_type = None
       for tag in w.tags:
-        if tag.k == 'name':
-          name = tag.v
         if tag.k == self.extract_type:
           way_type = tag.v
           should_extract = True
 
       if not should_extract:
         return
 
-      if not way_type in self.types:
+      if way_type not in self.types:
```

Comment:
Note that this isn't just a style change -- this actually changes the functionality. But I'm assuming the previous behavior was a bug here.